### PR TITLE
feat(iam-identity): adding trusted profiles

### PR DESCRIPTION
**Pull request by pooraniss-ibm**
## PR Checklist
Please make sure that your PR fulfills the following requirements:  
- [x] The commit message follows the [Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## Does this PR introduce a breaking change?    
- [ ] Yes
- [x] No

## Other information

Screenshot 2021-09-01 at 16 50 00

### CODE REVIEW

File: ibm_platform_services/iam_identity_v1.py [hunk 1]
```diff
@@ -40,7 +40,7 @@
 class IamIdentityV1(BaseService):
     """The iam_identity V1 service."""
 
-    DEFAULT_SERVICE_URL = 'https://iam.cloud.ibm.com'
+    DEFAULT_SERVICE_URL = 'https://iam.test.cloud.ibm.com'
```
Comment (Andras-Csanyi) [inline ibm_platform_services/iam_identity_v1.py:43]: The default service url always should point to the production environment.

Comment (padamstx) [reply ibm_platform_services/iam_identity_v1.py:43]: This most likely means that the "test" version of the API definition was used to generated the SDK code.  Need to be pretty careful about that :)

**Review by Andras-Csanyi — CHANGES_REQUESTED** [review]
The build is failing for some reason.

There are small nits to be fixed. Especially the `# begin/end...` tags. These annoying 
 lines keeps the public facing reference and the examples together. This is the reason behind the "mean" reviews.  :) 

Bottom line rules for these tags:
- they have to be consistent across SDKs, including singular/plural naming
- they are case sensitive

It seems this Python SDK is the one generated last, so I would use it as base line and align the other SDKs to this. Once it is done and the SDKs are merged in the apiref and example files have to be re-generated in order to keep the documentation updated.

File: ibm_platform_services/iam_identity_v1.py (hunk 1 above)
Comment (pooraniss-ibm) [reply ibm_platform_services/iam_identity_v1.py]: yes thats correct, but our staging and prod both endpoints work the same way, all the response model are same for both envs. however at one place I have missed to remove "test" :) (now corrected)

**Review by Andras-Csanyi — APPROVED** [review]
LGTM

**Pull request merged**